virttest/storage.py

```diff
@@ -79,8 +79,9 @@ def file_exists(params, filename_path):
         rbd_pool_name = params["rbd_pool_name"]
         rbd_image_name = "%s.%s" % (image_name.split("/")[-1], image_format)
         ceph_conf = params.get("ceph_conf")
+        keyring_conf = params.get("keyring_conf")
```

Comment:
Suggest the name `image_ceph_keyring_conf`

Reply:
Done